test/tests/functional/pbs_apbasil_inventory.py

```diff
@@ -43,34 +43,39 @@
 
 
 @tags('cray', 'mom')
-class TestBasilInventory(TestFunctional):
+class TestBasilQuery(TestFunctional):
```

Comment:
I see that the class changed because now the test does INVENTORY and SYSTEM queries.  That makes sense.  It's a little strange since the file name is pbs_apbasil_inventory.py.

Reply:
Done.